src/games/word-match.js

```diff
@@ -94,7 +103,7 @@ class WordMatch {
 
                 let resultsMatch = resultOneText.toLowerCase().trim() === resultTwoText.toLowerCase().trim();
                 if (resultsMatch) {
-                    let results = gameNode(colors.GREEN, null, {x: 50, y: 60}, {x: 20, y: 20}, {text: 'Same!', x: 50, y: 60});
+                    let results = gameNode(colors.GREEN, null, {x: 50, y: 60}, {x: 20, y: 20}, {text: "Same!", x: 50, y: 60});
```

Comment:
this game is unfinished (all of them are) so this will be cleaned up as I finish implementation